### PR TITLE
Release notes May

### PR DESCRIPTION
**Pull request by snowystinger**
Release notes for May release

### CODE REVIEW

File: packages/dev/docs/pages/releases/2023-05-19.mdx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: In this release, we are moving TagGroup and Column Resizing to GA. We also have some fixes to Calendars and have been busy making many improvements to the new React Aria Components we announced last release.
+date: 2023-05-19
```
Comment (LFDanLu) [inline packages/dev/docs/pages/releases/2023-05-19.mdx:15]: Just a reminder to update this date + header below + file name once we actually release

File: packages/dev/docs/pages/releases/2023-05-19.mdx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+# May 19, 2023 Release
+
+In this release, we are moving TagGroup and Column Resizing to GA. We have also been making many improvements and bug fixes to the new [React Aria Components](../react-aria/react-aria-components.html) we announced last release. We have also added the much asked for Search to our docs. We hope this will aid in discoverability for our components and hooks.
+
+As always, thanks to all of our contributors for helping to make this release possible! Special shoutout to [@auskast](https://github.com/auskast) for an extensive PR to help us improve our React Aria Components.
+
+## New Components
+
+- [TagGroup](https://react-spectrum.adobe.com/react-spectrum/TagGroup.html)
```
Comment (LFDanLu) [inline packages/dev/docs/pages/releases/2023-05-19.mdx:26]: I know tableview isn't new and that it is mentioned in the description above, but do we wanna explicitly mention TableView column resizing GA in this section w/ a link to the section in the spectrum/aria docs as well?

File: packages/dev/docs/pages/releases/2023-05-19.mdx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+### TagGroup API changes in official release
```
Comment (LFDanLu) [inline packages/dev/docs/pages/releases/2023-05-19.mdx:28]: Once the PR with these changes go in, would be good to link to it here: https://github.com/adobe/react-spectrum/pull/4548/. Maybe we can forgo these lines in favor of pointing to the PR instead as well?

**Review by LFDanLu — COMMENTED** [review]
Will finish auditing the PRs listed tmrw, just wanted to list some reminders/questions first

File: packages/dev/docs/pages/releases/2023-05-19.mdx (hunk 1 above)
Comment (snowystinger) [reply packages/dev/docs/pages/releases/2023-05-19.mdx:15]: yep, i set it to tomorrow, but it's possibly a little optimistic, haha

File: packages/dev/docs/pages/releases/2023-05-19.mdx (hunk 2 above)
Comment (snowystinger) [reply packages/dev/docs/pages/releases/2023-05-19.mdx:26]: Yeah, i can do that

File: packages/dev/docs/pages/releases/2023-05-19.mdx (hunk 3 above)
Comment (snowystinger) [reply packages/dev/docs/pages/releases/2023-05-19.mdx:28]: I like making the necessary changes very visible and easy to find for people. I don't want to hide them behind a link

Comment (LFDanLu) [reply packages/dev/docs/pages/releases/2023-05-19.mdx:28]: Makes sense

**Review by devongovett — APPROVED** [review]

**Pull request merged**